### ISSUE TITLE
Track last vote time across reconnections

### ISSUE THREAD

**Issue by gyunderscorebe** [ActionFPS/ActionFPS-Game#130]
# Feature

## What?

Save time when a player last voted on disconnect, restore it on reconnection.

## Why?

This time is used in vote spam detection. Implementing this would prevent two common "trolling" methods:

1. A troll keeps calling a bogus vote, reconnecting when it fails and calling again.
2. A troll keeps calling a vote, immediately disconnecting (canceling the vote) and reconnecting to vote again. This variant is a lot more noisy and it's difficult to ban someone who keeps reconnecting.

This depends on #129 as reconnecting is the only way to work around that issue.

**Comment by ScalaWilliam** [comment]
PS we'll have fewer trolls with the auth system ;)

**Comment by ScalaWilliam** [comment]
Making postponed.
